content/docs/csidriver/installation/offline/_index.md

```diff
@@ -225,9 +225,9 @@ Preparing a offline bundle for installation
    dellemc/csi-vxflexos:v2.7.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.7.0
    dellemc/csi-vxflexos:v2.8.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.8.0
    dellemc/csi-vxflexos:v2.9.0 -> localregistry:5000/csi-operator/csi-vxflexos:v2.9.0
-   dellemc/sdc:3.6 -> localregistry:5000/csi-operator/sdc:3.6
    dellemc/sdc:3.6.0.6 -> localregistry:5000/csi-operator/sdc:3.6.0.6
```

Comment:
What about the removal of the other dellemc/sdc:* values?

Reply:
Done.